### PR TITLE
Fix prefix in test files

### PR DESCRIPTION
**Pull request by maddieclayton**
## Description
Fix issue: https://github.com/Azure/azure-powershell/issues/8124
Common pr: https://github.com/Azure/azure-powershell-common/pull/120

## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by markcowl** [comment]
@maddieclayton Why do we need to do this?  We should certainly enable folks to use either prefix, but I don't know that we shoudl change this wholesale

**Comment by maddieclayton** [comment]
@markcowl I thought it would be good for all of the tests to have the Az prefix so that partners coming in would see that they should use the Az prefix across the board for tests.  I also added some tests for Enable/Disable-AzureRmAlias, so we don't need to use aliases to test those cmdlets anymore.

File: src/LogicApp/LogicApp.Test/ScenarioTests/WorkflowController.cs [hunk 1]
```diff
@@ -90,7 +90,7 @@ public void RunPowerShellTest(XunitTracingInterceptor logger, params string[] sc
                     "ScenarioTests\\Common.ps1",
                     "ScenarioTests\\" + callingClassName + ".ps1",
                     _helper.RMProfileModule,
-                    _helper.GetRMModulePath(@"AzureRM.LogicApp.psd1"),
+                    _helper.GetRMModulePath(@"Az.LogicApp.psd1"),
```
Comment (MiYanni) [inline src/LogicApp/LogicApp.Test/ScenarioTests/WorkflowController.cs]: I don't think we support this yet. AFAIK, it still needs to be `"AzureRM.LogicApp.psd1"` for it to resolve correctly in `GetRMModulePath`.

Comment (MiYanni) [reply src/LogicApp/LogicApp.Test/ScenarioTests/WorkflowController.cs]: Discussed offline. Should work with `"Az.LogicApp.psd1"`.

**Pull request merged**